examples/Catena4430_Sensor/Catena4430_cMeasurementLoop.cpp

```diff
@@ -265,6 +265,7 @@ cMeasurementLoop::fsmDispatch(
             newState = State::stRebootForUpdate;
         else
             newState = State::stSleeping;
+        this->handleSdTTNv3Migrate();
```

Comment:
This should not really be here. We're combining things. We need a separate state for Migrating, and we need to be very clear (in the state diagram) how we handle the combination of firmware update and migration. This mixes them up and makes it unclear.

Reply:
https://github.com/mcci-catena/mcci-catena-4430/blob/68b3e7b66b1c2660f761e6f77fb8b7fc29bdb51e/examples/Catena4430_Sensor/Catena4430_cMeasurementLoop.cpp#L263-L274